authui/src/colorscheme.ts

```diff
@@ -1,43 +1,31 @@
-import { Controller } from "@hotwired/stimulus";
-
-export class ColorSchemeController extends Controller {
-  static values = {
-    darkThemeEnabled: Boolean,
-  };
-
-  queryResult = window.matchMedia("(prefers-color-scheme: dark)");
-
-  declare darkThemeEnabledValue: Boolean;
-
-  onChange = () => {
-    if (!this.darkThemeEnabledValue) {
-      return;
-    }
-
-    let explicitColorScheme = "";
-    const metaElement = document.querySelector("meta[name=x-color-scheme]");
-    if (metaElement instanceof HTMLMetaElement) {
-      explicitColorScheme = metaElement.content;
-    }
+function onChange() {
+  const htmlElement = document.getElementsByTagName("html")[0];
```

Comment:
`document.documentElement` is the [HTMLHtmlElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHtmlElement) you want.